app/backend/src/couchers/servicers/requests.py

```diff
@@ -164,6 +177,43 @@ def CreateHostRequest(self, request, context, session):
         if to_date - from_date > timedelta(days=365):
             context.abort(grpc.StatusCode.INVALID_ARGUMENT, errors.DATE_TO_AFTER_ONE_YEAR)
 
+        # Check if user has been sending host requests excessively
+        count_host_requests_last_24h = session.execute(
+            select(func.count())
+            .select_from(HostRequest)
+            .join(Conversation, HostRequest.conversation_id == Conversation.id)
+            .where(HostRequest.surfer_user_id == context.user_id)
+            .where(Conversation.created >= now() - timedelta(hours=24))
+        ).scalar_one()
+        if count_host_requests_last_24h > HOST_REQUEST_DAILY_BAN_QUOTA - 2:
```

Comment:
So since the user isn't actually stopped from sending more reqs, this will send up to ~`HOST_REQUEST_SPAM_USER_BAN-HOST_REQUEST_DAILY_BAN_QUOTA` to the mods, right?

Reply:
No, inside the action is aborted with an error, and the user is banned, so they can't send more. But this is also a draft, not sure if this should be final - waiting for yours and Jesses opinions still :)

Reply:
I don’t think we should ever auto-ban, especially without warning. Let’s discuss on slack.

Reply:
Removed the banning - only blocking further requests now 👍